### PR TITLE
[WIP] statsmodels and anaomalydetection flux packages

### PR DESCRIPTION
**Pull request by Anaisdg**
### Done checklist
- [ ] docs/SPEC.md updated
- [ ] Test cases written

### CODE REVIEW

File: stdlib/contrib/anaisdg/anomalydetection/mad.flux [hunk 1]
```diff
@@ -0,0 +1,40 @@
+package anomalydetection 
+
+import "math"
+import "experimental"
+
+mad = (table=<-, threshold) => {
```
Comment (sanderson) [inline stdlib/contrib/anaisdg/anomalydetection/mad.flux:6]: Should users always have to specify the `threshold` or is there a reasonable default?

File: stdlib/contrib/anaisdg/anomalydetection/mad.flux [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+    // _value_diff = xi-med(xi)
+    diff = experimental.join(
+    left: data,
+    right: med,
+    fn: (left, right) => ({ right with _value: math.abs(x: left._value - right._value) })
+    )
+   
+   //The constant b is needed to make the estimator consistent for the parameter of interest.
+    /// In the case of the usual parameter a at Gaussian distributions
+    b = 1.4826
```
Comment (sanderson) [inline stdlib/contrib/anaisdg/anomalydetection/mad.flux:20]:
```
// The constant b makes the estimator consistent for the parameter of interest.
// In the case of the usual parameter at Gaussian distributions
b = 1.4826
```

I'm not sure what that 2nd line in the comment is trying to say, but it seems like an incomplete sentence.

File: stdlib/contrib/anaisdg/anomalydetection/mad_test.flux [hunk 3]
```diff
@@ -0,0 +1,79 @@
+package anomalydetection_test
+
+import "testing"
+import "anomalydetection"
```
Comment (sanderson) [inline stdlib/contrib/anaisdg/anomalydetection/mad_test.flux:4]: You need the full import path: `contrib/anaisdg/anomalydetection`

File: stdlib/contrib/anaisdg/anomalydetection/mad_test.flux [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+,,4,14.184349556083532,2020-05-06T16:13:40Z,0.6744907594765952,normal
+"
+
+t_mad = (table=<-) =>
+	table
+		|> range(start: 2020-04-29T21:04:00Z, stop: 2020-05-29T21:05:00Z)
+		|> anomalyDetection.mad(threshold: 3.0)
+
+
+test _linearRegression = () =>
```
Comment (sanderson) [inline stdlib/contrib/anaisdg/anomalydetection/mad_test.flux:76]: Update the test name to `_mad`

File: stdlib/contrib/anaisdg/statsmodels/linearreg_test.flux [hunk 5]
```diff
@@ -0,0 +1,37 @@
+package statsmodels_test
+
+import "testing"
+import "statsmodels"
```
Comment (sanderson) [inline stdlib/contrib/anaisdg/statsmodels/linearreg_test.flux:4]: You need the full import path: `contrib/anaisdg/statsmodels`

File: stdlib/contrib/anaisdg/anomalydetection/mad.flux (hunk 2 above)
Comment (Anaisdg) [reply stdlib/contrib/anaisdg/anomalydetection/mad.flux:20]: The complete sentence is "In the case of the usual parameter at Gaussian distributions
b = 1.4826"
Do you have a suggestion for how I should change it? Does this read better? 
"In the case of the usual parameter at Gaussian distributions, " 
b = 1.4826
Or should I just do: 
"In the case of the usual parameter at Gaussian distributions b = 1.4826" 
Thank you

Comment (sanderson) [reply stdlib/contrib/anaisdg/anomalydetection/mad.flux:20]: Oh I see. The variable declaration is meant to be read as a continuation of the comment. I think that's what threw me. I'm used to comments being complete ideas without relying on the accompanying source code. I don't know that it's wrong; just not something I'm used to.

File: stdlib/contrib/anaisdg/anomalydetection/mad.flux (hunk 1 above)
Comment (Anaisdg) [reply stdlib/contrib/anaisdg/anomalydetection/mad.flux]: There is a reasonable default. threshold = 3.0. Thank you.

**Pull request closed without merging**